trash.go

```diff
@@ -487,10 +492,12 @@ func removeEmptyDirs(rootPackage string) error {
 	return nil
 }
 
-func cleanup(dir string) error {
-	gopath := path.Join(dir, "..", "..", "..", "..")
-	gopath = filepath.Clean(gopath)
-	os.Setenv("GOPATH", gopath)
+func cleanup(dir string, trashDir string, clearCache bool) error {
+    	cacheFolder := os.Getenv("GOPATH")
```

Comment:
This really makes no sense to me: cacheFolder is **not** the GOPATH: not in the cleanup stage. Even if it was, you don't need to pass it through environment. BTW, you've just added `trashDir` argument which is not used: why?

At this point we're assuming the project's GOPATH, hence the `../../../..`